bedrock/firefox/templates/firefox/nightly_whatsnew.html

```diff
@@ -0,0 +1,108 @@
+        </header>
+        <div class="main-content">
+          <h1>{{ self.page_title() }}</h1>
+          <h2>
+            {% trans %}
+              Every 6 to 8 weeks, a new major version of Firefox is released and as a result, Nightly gets its version
+              number increased as well.
+            {% endtrans %}
+          </h2>
+          <div class="body">
+            <p>
+              {% trans %}
+                This is a good time to thank you for helping us make Firefox better and to give you some pointers to
+                documentation, communication channels and news sites related to Firefox Nightly that may be of interest
+                to you.
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans blog='https://blog.nightly.mozilla.org/', twitter='https://twitter.com/FirefoxNightly' %}
+                If you want to know what’s happening around Nightly and its community, reading our
+                <a href="%(blog)s">blog</a> and following us on <a href="%(twitter)s">Twitter</a> are good starting
+                points!
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans mdn='https://developer.mozilla.org/en-US/Firefox/Experimental_features'
+                           '?utm_source=firefox&utm_medium=whatsnew&utm_campaign=nightly' %}
+                If you are interested in knowing which platform features you could test on Nightly and can’t see yet on
+                other Firefox channels, then have a look at the <a href="%(mdn)s">Experimental Features</a> page on
+                <abbr title="Mozilla Developer Network">MDN</abbr>.
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans bugzilla='https://bugzilla.mozilla.org' %}
+                Do you experience crashes? Unexpected behaviors? Web compatibility regressions? We’d love to get them
+                filed in <a href="%(bugzilla)s">Bugzilla</a> to make sure they don’t make it to the final release (extra
+                karma if you add the <em>nightly-community</em> keyword to your bug reports)!
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans triagebot='https://triagebot.mozilla.org/' %}
+                Would you be willing to help us triage unconfirmed bugs in Bugzilla? Then maybe
+                <a href="%(triagebot)s">TriageBot</a> could be of interest to you.
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans irc='ircs://irc.mozilla.org/nightly' %}
+                You can also interact with the Nightly community on IRC in the <a href="%(irc)s">#nightly</a> channel,
+                feel free to ask questions and talk about features, regressions or crashes you may be experiencing.
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans %}
+                <strong>Go community!</strong>
```

Comment:
Should probably put the `strong` outside the `trans` so the HTML isn't part of the string.